insights/tests/client/connection/test_checkin.py

```diff
@@ -24,6 +24,7 @@ def _get_canonical_facts_response(canonical_facts):
 @patch('insights.client.connection.InsightsUploadConf.get_rm_conf', return_value={})
 @patch(
     "insights.client.connection.InsightsConnection._init_session", Mock())
+@patch("insights.client.connection.InsightsConnection._fetch_system_by_machine_id", return_value=True)
```

Comment:
I always tried to keep the order of patches matching the order of first call in the production code. In this case, `_fetch_system_by_machine_id` is the first thing that is called in the tested `checkin` method.

As I see, however, the order of the existing patches doesn’t follow this rule: `post` is called after `get_canonical_facts`. 🤷🏻‍♂️ So no need to worry, just noting.

```python
@patch(
    "insights.client.connection.InsightsConnection.post",
    **{"return_value.status_code": codes.CREATED}
)
@patch(
    "insights.client.connection.get_canonical_facts",
    return_value=_get_canonical_facts_response({"subscription_manager_id": str(uuid4())})
)
@patch("insights.client.connection.InsightsConnection._fetch_system_by_machine_id", return_value=True)
```